wca-guidelines.md

```diff
@@ -205,10 +205,15 @@ To be more informative, each Guideline is classified using one of the following
-- E2c++) [CLARIFICATION] The competitor's solution must only use moves that are exactly defined in [Regulation 12a](regulations:regulation:12a). Examples of notation and moves that are not permitted in competition: [F], [R, U], [R: U], R'2, L'w, f, M, U2'.
-- E2c+++) [ADDITION] If the competitor forgot to write their name on their intended solution paper before the time limited, they are permitted to add their name in front of a judge. In this case, the competitor must ask the judge while submitting their solution and then immediately write their name (and make no other changes) under direct supervision of the judge. Once a competitor has finished submitting their solution (e.g. placing it in the judge's hands and allowing the judge to move on to the next competitor, placing it on the judge's table and walking away), they are not permitted to add their name. If the competitor writes their name after the time limit but before asking the judge, the attempt remains disqualified (DNF).
+- E2c+) [CLARIFICATION] Although the judge may provide a standard sheet for competitors to submit solutions, a competitor may submit a solution on a different piece of paper. (Note that the piece of paper must come from the judge, according to [Regulation E3a](regulations:regulation:E3a).)
+- E2c1++) [EXPLANATION] The competitor is not required to submit a paper with the competition name, round, and/or attempt number, but may wish to do so in order to help the organization team keep track of submitted solutions.
+- E2c1+++) [ADDITION] If a judge finds a solution without a information to identify the attempt after the competitor has finished submitting it (e.g. while grading the attempt), they should find out the competitor who wrote the solution in order to assign them a DNF (rather than DNS). Note that the competitor must not be allowed to write information to identify the attempt at this point, so they must receive a DNF result.
+- E2c2+) [EXAMPLE] Examples that are not considered a single unambiguous move sequence: multiple solution sequences without exactly one clearly indicated solution, a solution with ambiguous symbols (e.g. a letter that looks similar to "B" and "R" but is not clearly one or the other), a solution with stars or long arrows indicating moves to be performed out of written order (e.g. "insertions" that are not written in-line with the rest of the move sequence, "pre-moves" that are not incorporated into the move sequence using valid notation), a move sequence that is not written out roughly as a series of lines in reading order, any arrangement of symbols that cannot be interpreted as an unambiguous move sequence. Exceptions to the examples above: small typographic corrections (e.g. blacking out a move and writing a move above that is clearly meant to replace it, moves written in subscript or superscript to fit between other moves).
```

Comment:
> Do we want the `moves written in subscript or superscript to fit between other moves`?

These are intended to cover the cases where a competitor doesn't *quite* write things in lines, but does something that is clearly intended to be read that way: https://github.com/thewca/wca-regulations/pull/575#issuecomment-407871264

After finishing the 2019 draft I want to work on an FMC [guide](https://github.com/thewca/wca-regulations/issues/236), where I want to include examples like the ones more clear.

> 1. What is a _long_ arrow? Are insertions written with a _short_ arrow fine?

Yeah, the intention is to allow simple things like in your image (where moves aren't exactly "written out of order, but definitely use small arrows). However, I wrote this before I added the "small typographic corrections" part, which I believe covers it. I've taken out "long".

> 2\. I think not all Delegates know what exactly an insertion is or how it looks like, so this might not be applied consistently.

I would like to leave this in, since it's a well-known term in most of the community (similar to "stickerless") and makes the intent pretty clear.

Reply:
I still don't like this and I think this will be interpreted inconsistently.
What is a "small typographic correction"? What if I made a mistake when copying my solution and accidentally skipped 4 moves? Is this still small?
And how can the judge make sure if it's an insertion or not (especially relevant when the insertion is a sexy move or something like `R U2 R'`).

Reply:
An insertion is fine if it's in-line, and I think most typographic corrections are clear in practice.

Do you have alternative suggestions for how to express that the competitor is responsible for writing a single move sequence in order, while still being lenient to the examples you gave in the other draft PR?